### ISSUE TITLE
El código no tiene sentido

### ISSUE THREAD

**Issue by TH3AL3X** [Hacking-pch/WHATSAPP-BANEO#3]
E deobfuscado el código con un echo, y él código shell y php no tiene sentido, ya qué no guarda el número de teléfono por ningún lado, código sin sentido y sin funcionalidad :-1:

**Comment by TH3AL3X** [comment]
[script.txt](https://github.com/Hacking-pch/WHATSAPP-BANEO/files/5614494/script.txt)
Es un intento fallido de hacer un request post, no se porque la gente usa esto cuando no hace nada lmfao...

**Comment by 0x14Rp** [comment]
😂😂

**Comment by Neptune650** [comment]
Es cierto, únicamente es un código que no funciona y ofuscado para atraer a estúpidos.
Porque lo pone en GitHub si lo va a ofuscar? 😂

**Comment by Isra16k** [comment]
Hay alguno que si funcione?

**Comment by Neptune650** [comment]
> Hay alguno que si funcione? 

No, ninguno de estos "hacks" te va a servir para algo.

**Comment by Isra16k** [comment]
> > Hay alguno que si funcione? 
> 
> No, ninguno de estos "hacks" te va a servir para algo.

¿O existirá algún método para banear cuenta? Necesito ayuda hay un sujeto que ya nos tiene hartos

**Comment by Neptune650** [comment]
> > > Hay alguno que si funcione? 
> > 
> > No, ninguno de estos "hacks" te va a servir para algo.
> 
> ¿O existirá algún método para banear cuenta? Necesito ayuda hay un sujeto que ya nos tiene hartos

No, si tienes algún problema grave con un usuario contacta el soporte de WhatsApp.

**Comment by TH3AL3X** [comment]
> > > Hay alguno que si funcione? 
> > 
> > No, ninguno de estos "hacks" te va a servir para algo.
> 
> ¿O existirá algún método para banear cuenta? Necesito ayuda hay un sujeto que ya nos tiene hartos

Yo tengo mi propio método y es muy sencillo...

**Comment by Isra16k** [comment]
> > > > Hay alguno que si funcione? 
> > > 
> > > No, ninguno de estos "hacks" te va a servir para algo.
> > 
> > ¿O existirá algún método para banear cuenta? Necesito ayuda hay un sujeto que ya nos tiene hartos
> 
> Yo tengo mi propio método y es muy sencillo...

¿Podrías ayudarme?
